uber/tests/models/test_utils.py

```diff
@@ -93,3 +93,38 @@ def test_charge_log_transaction_no_model(self):
         charge = Charge(amount=1000, description="Test charge")
         Charge.charge_cc(charge, Mock(), 1)
         assert not Charge.stripe_transaction_from_charge.called
+
+
+class TestAgeCalculations:
+
+    @pytest.mark.parametrize('birthdate,today,expected', [
+        # general
```

Comment:
These DOBs and their results will quickly become outdated. You probably want to instead store an intended timedelta (years+months+days) and then pass localized_now() minus the timedelta through to the function for testing.

Reply:
I was actually thinking about this and I was trying to figure out whether this was a problem. I hope you don't mind if I push back a little on this one.

Some of these cases could use a timedelta in that way because they pertain to a specific span of time, and start/end dates aren't important. But most of the test cases pertain to a specific type of date rather than a timespan: they start or end on a leap year, or on Feb 29.

For those test cases, you could only shift the year by increments of four, and that would just be a translation to an equivalent test case. For example, a test case of `2000-1-1` to `2018-1-4` should be equivalent to a test case of `2028-1-1` to `2046-1-4`, or even `2160-1-1` to `2178-1-4`, as long as they all start on a leap year and span the same time frame.

My understanding is that these hard-coded dates should continue to make valid test cases for a developer in the future. I'm open to changing some/all of these test cases per your suggestion, but I don't know right now how I should implement that, or what problem that it would solve.

Reply:
Whoops, no, I just can't read. I didn't notice you were also passing in a date for "today." These are fine!